<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>topNav</class>
 <widget class="QWidget" name="topNav">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>78</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#3f51b5;
border:1px solid #3f51b5;
max-width:35px;

}
QPushButton:hover{
border:2px solid #b3d4fc;

        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
        border-bottom-right-radius: 4px;
        border-bottom-left-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/ic_help_outline_white_2x.png</normaloff>:/Icons/ic_help_outline_white_2x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>SFMono Nerd Font Mono</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color:#3f51b5;
border:1px solid #3f51b5;
color:white;
font-size:14px;
height:200px;
margin:0px 0px 0px 0px;
}</string>
       </property>
       <property name="text">
        <string>مكتبة و وراقة</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_logOut">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#3f51b5;
border:1px solid #3f51b5;
max-width:35px;

}
QPushButton:hover{
border:2px solid #b3d4fc;

        border-top-left-radius: 4px;
        border-top-right-radius: 4px;
        border-bottom-right-radius: 4px;
        border-bottom-left-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/ic_exit_to_app_white_2x.png</normaloff>:/Icons/ic_exit_to_app_white_2x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="BT_Books">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
 position: relative;
    padding: 0px 0px;
        padding-top: 0px;
        padding-bottom: 0px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #9E9E9E;
    white-space: nowrap;
    background-color: #3F51B5;
   
    border-radius: 0px;
	  border: 1px solid #3F51B5 ;
    
      
        
    vertical-align: middle;
}

QPushButton:hover
{
 background-color: #03A9F4;
 border-top: 1px solid  #03A9F4 ;
 border-left: 1px solid  #03A9F4 ;
 border-right: 1px solid  #03A9F4 ;
border-bottom: 2px solid #FF4081;
 color: #ffffff;

}
</string>
       </property>
       <property name="text">
        <string>الكتب / الوثائق</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/ic_attachment_white_24dp.png</normaloff>:/Icons/ic_attachment_white_24dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_Workers">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
 position: relative;
    padding: 0px 0px;
        padding-top: 0px;
        padding-bottom: 0px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #9E9E9E;
    white-space: nowrap;
    background-color: #3F51B5;
   
    border-radius: 0px;
	  border: 1px solid #3F51B5 ;
    
      
        
    vertical-align: middle;
}

QPushButton:hover
{
 background-color: #03A9F4;
 border-top: 1px solid  #03A9F4 ;
 border-left: 1px solid  #03A9F4 ;
 border-right: 1px solid  #03A9F4 ;
border-bottom: 2px solid #FF4081;
 color: #ffffff;

}
</string>
       </property>
       <property name="text">
        <string>المـوظفين</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/ic_work_white_24dp.png</normaloff>:/Icons/ic_work_white_24dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_Clients">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
 position: relative;
    padding: 0px 0px;
        padding-top: 0px;
        padding-bottom: 0px;
    font-size: 15px;
    font-family: Ubuntu;
    color: #9E9E9E;
    white-space: nowrap;
    background-color: #3F51B5;
   
    border-radius: 0px;
	  border: 1px solid #3F51B5 ;
    
      
        
    vertical-align: middle;
}

QPushButton:hover
{
 background-color: #03A9F4;
 border-top: 1px solid  #03A9F4 ;
 border-left: 1px solid  #03A9F4 ;
 border-right: 1px solid  #03A9F4 ;
border-bottom: 2px solid #FF4081;
 color: #ffffff;

}
</string>
       </property>
       <property name="text">
        <string>الزبائــن</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/Icons/ic_supervisor_account_white_24dp.png</normaloff>:/Icons/ic_supervisor_account_white_24dp.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
